### ISSUE TITLE
Enable WireMock ResponseTemplates

### ISSUE THREAD

**Issue by BartRobeyns** [authorjapps/zerocode#257]
The ResponseTemplates allow us to add simple manipulations to the response, making the test-cases more dynamic.
Especially for testing #246 (retry on REST-requests), in combination with PR #249  (date.after and date.before assertions), using {{now offset='-1 minutes'}} would allow us to mimic the retry based on a date in the response that after a couple of tries reaches the date in the assertion.

**Comment by santhoshTpixler** [comment]
@BartRobeyns  Kindly reference the issue or pull request compatible with GitHub. 
Refer: https://help.github.com/en/articles/autolinked-references-and-urls
Proper referencing will come in handy.

**Comment by authorjapps** [comment]
PR for this is https://github.com/authorjapps/zerocode/pull/259 which is being reviewed.

@BartRobeyns , Thanks for the PR . 

We are goign through the PR.
Just for everyone's benefit , can you please put some light on the below.
_(Also it will help the users to understand its usage)_
```
"body":{
    "fixed_date_time": "${LOCAL.DATETIME.NOW:uuuu-MM-dd HH:mm:ss.SSS}",
    "dynamic_oneMinuteAgo": "{{now offset='-1 minutes' format='yyyy-MM-dd HH:mm:ss.SSS'}}",
    "dynamic_oneMinuteAhead": "{{now offset='+1 minutes' format='yyyy-MM-dd HH:mm:ss.SSS'}}",
    "ucparam": "{{upper request.query.param[0]}}"
}

```
from the PR [file](https://github.com/authorjapps/zerocode/pull/259/files#diff-4221aab085d448fba1cb91c5a0610fe9)

Also
```
            "assertions": {
                "status": 200,
                "body": {
                    "ucparam": "LOWERTOUPPER"
                    // Once PR#249 is accepted, "fixed_date_time": "$DATE.AFTER${$.GetTemplate.response.dynamic_oneMinuteAgo}"
                    // and "fixed_date_time": "$DATE.BEFORE${$.GetTemplate.response.dynamic_oneMinuteAhead}"
                    // should work
                }
            }
```

Also we will capture these into the README file 👍

**Comment by BartRobeyns** [comment]
I've added some explanation for the 'body'-part.
If you can tell me what's not clear from the assertions-part, i'll gladly try to explain it.

**Comment by authorjapps** [comment]
> // Once PR#249 is accepted, "fixed_date_time": "$DATE.AFTER${$.GetTemplate.response.dynamic_oneMinuteAgo}"

Makes sense! #249 has been merged now. fyi only.

**Comment by authorjapps** [comment]
Feature available in version `1.3.9`
